SDK/simVis/EphemerisVector.cpp

```diff
@@ -172,7 +173,12 @@ void EphemerisVector::rebuild_(const simData::PlatformPrefs& prefs)
   // Draw the moon vector
   if (prefs.drawmoonvec())
   {
+#if OSGEARTH_VERSION_LESS_THAN(2,10,0)
```

Comment:
Current osgEarth version is 2.9.0.  Just pulled the latest master and it's still listed as 2.9.0, DEVEL = 0.  I think this requires an update to osgEarth/Version to work

Reply:
Yep, forgot to push the Version bump... now done.